bake

```diff
@@ -540,18 +540,17 @@ function bake_update () {
 
 function bake_create_bakefile () {
   if [ ! -e "Bakefile" ]; then
-    cat > "Bakefile" <<END
+    cat > "Bakefile" <<'END'
 #!/usr/bin/env bash
 
 # bake_require github.com/kyleburton/bake-recipes/docker/docker.sh
 
+# Set default editor if not defined
+export EDITOR="${EDITOR:-vi}"
+
 bake_task get-started "Get started by editing this Bakefile"
 function get-started () {
-  if [ -n "$EDITOR" ]; then
     "$EDITOR" Bakefile
```

Comment:
would you kindly include an else that echo's a message?  the task description indicates that it will bring up an editor.

Reply:
I set the default to `vi` in line 549, so there won't be an instance where it's not set.

```
export EDITOR="${EDITOR:-vi}"
```

Reply:
check, ty!